### ISSUE TITLE
Not displaying error messages in Goland on macOS

### ISSUE THREAD

**Issue by tobinguyenn** [xxpxxxxp/intellij-plugin-golangci-lint#155]
**Description of the problem**
Not displaying error messages when using Goland on macOS Sonoma (IDE doesn't show any plugin errors or something)., but `golangci-lint` run normally in terminal. Still working with my VS Code and same setting on Ubuntu

**OS version**
```
Sonoma 14.2 (23C64)
```

**Goland version**
```
Goland 2023.3.2
```

**Version of golangci-lint**
```console
$ golangci-lint --version
golangci-lint has version 1.55.2 built with go1.21.3 from e3c2265f on 2023-11-03T12:59:25Z
```

**.golang-ci.yml**
[golangci.txt](https://github.com/xxpxxxxp/intellij-plugin-golangci-lint/files/13831493/golangci.txt)

**Go environment**
```console
$ go env
GO111MODULE=''
GOARCH='arm64'
GOBIN=''
GOCACHE='/Users/tobi/Library/Caches/go-build'
GOENV='/Users/tobi/Library/Application Support/go/env'
GOEXE=''
GOEXPERIMENT=''
GOFLAGS=''
GOHOSTARCH='arm64'
GOHOSTOS='darwin'
GOINSECURE=''
GOMODCACHE='/Users/tobi/go/pkg/mod'
GONOPROXY=''
GONOSUMDB=''
GOOS='darwin'
GOPATH='/Users/tobi/go'
GOPRIVATE=''
GOPROXY='https://proxy.golang.org,direct'
GOROOT='/usr/local/go'
GOSUMDB='sum.golang.org'
GOTMPDIR=''
GOTOOLCHAIN='auto'
GOTOOLDIR='/usr/local/go/pkg/tool/darwin_arm64'
GOVCS=''
GOVERSION='go1.21.4'
GCCGO='gccgo'
AR='ar'
CC='clang'
CXX='clang++'
CGO_ENABLED='1'
GOMOD='/Users/tobi/Work/personal/learning/practical-go-lessons/go.mod'
GOWORK=''
CGO_CFLAGS='-O2 -g'
CGO_CPPFLAGS=''
CGO_CXXFLAGS='-O2 -g'
CGO_FFLAGS='-O2 -g'
CGO_LDFLAGS='-O2 -g'
PKG_CONFIG='pkg-config'
GOGCCFLAGS='-fPIC -arch arm64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -ffile-prefix-map=/var/folders/rn/8bcyk4895_3gqm8t2jzk9ntw0000gp/T/go-build1807327111=/tmp/go-build -gno-record-gcc-switches -fno-common'
```

**File watcher**
<img width="1144" alt="Screenshot 2024-01-04 at 22 07 11" src="https://github.com/xxpxxxxp/intellij-plugin-golangci-lint/assets/30967513/60fee36d-03c4-4553-8b4e-caa42f7b67d8">

**Go linter**
<img width="979" alt="Screenshot 2024-01-04 at 22 07 48" src="https://github.com/xxpxxxxp/intellij-plugin-golangci-lint/assets/30967513/d99f91ac-6466-4c25-af66-7b86d0403e2d">

**Comment by vitorvasc** [comment]
I'm facing the same issue but I still have the `Ventura 13.6.3` version installed here, not sure if this is an issue with the OS version.

**Comment by xxpxxxxp** [comment]
Hi @tobinguyenn , would you help search `go-linter` in Goland log, if any error log please paste it in the thread. The Goland log can be found by clicking `Help` -> `Show log in Finder` -> open it with an editor.

**Comment by tobinguyenn** [comment]
@vitorvasc @xxpxxxxp 
I just discovered that the issue is caused by the Go Linter pointing to the wrong path for the `.golangci.yml` file. I changed the name of my project folder from ~/Users/$NAME/**w**ork... to ~/Users/$NAME/**W**ork... (I capitalized the 'w'), but the Go linter didn't recognize the new path, it still uses the old path. I tried changing it to a completely different folder name and reloading the IDE, and everything worked normally.
